### PR TITLE
Update Node test versions

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,11 @@
 language: node_js
 
 node_js:
-  - '8'
-  - '7'
-  - '6'
-
-before_install:
```
Comment (karellm) [inline .travis.yml]: Why did you delete these lines?

Comment (MichaelDeBoey) [reply .travis.yml]: Travis automatically [uses the latest version of each major we define](https://travis-ci.org/i18next/i18next-parser/jobs/636452976#L181) and [logs it](https://travis-ci.org/i18next/i18next-parser/jobs/636452976#L204) too

Comment (MichaelDeBoey) [reply .travis.yml]: @karellm Travis automatically [uses the latest version of each major we define](https://travis-ci.org/i18next/i18next-parser/jobs/636452976#L181) and [logs it](https://travis-ci.org/i18next/i18next-parser/jobs/636452976#L204) too

**Comment by MichaelDeBoey** [comment]
Ping @karellm

**Comment by karellm** [comment]
Thanks, I merged it
